### PR TITLE
hostapd:  ft_psk_generate_local and ft_over_ds support

### PR DESCRIPTION
**Pull request by BigNerd95**
Add support for ft_psk_generate_local and ft_over_ds flags in ieee80211r

Signed-off-by: Lorenzo Santina <lorenzo.santina@edu.unito.it>

### CODE REVIEW

File: package/network/services/hostapd/files/hostapd.sh [hunk 1]
```diff
@@ -392,13 +392,15 @@ hostapd_set_bss_options() {
 			set_default reassociation_deadline 1000
 			set_default pmk_r1_push 0
 			set_default ft_psk_generate_local 0
+			set_default ft_over_ds 0
```
Comment (hauke) [inline package/network/services/hostapd/files/hostapd.sh]: The hostapd documentation says this is activated by default:
```
# Whether to enable FT-over-DS
# 0 = FT-over-DS disabled
# 1 = FT-over-DS enabled (default)
#ft_over_ds=1
```
Why do you deactivate it by default now?

**Comment by BigNerd95** [comment]
Sorry, my mistake
Just fixed

I tested this on TP-LINK WR841ND v8 and v10

**Comment by hauke** [comment]
Thank you for the patch, it was added to master.

**Comment by BigNerd95** [comment]
I think you can close 
https://github.com/lede-project/source/pull/1212

**Pull request closed without merging**